gdal/ogr/ogrsf_frmts/lvbag/ogrlvbagdriver.cpp

```diff
@@ -38,13 +38,14 @@ static int OGRLVBAGDriverIdentify( GDALOpenInfo* poOpenInfo )
     if( !poOpenInfo->bStatOK )
         return FALSE;
     if( poOpenInfo->bIsDirectory )
-        return -1;  // Unsure.
+        return -1;  // Check later
     if( poOpenInfo->fpL == nullptr )
         return FALSE;
  
-    CPLString osExt(CPLGetExtension(poOpenInfo->pszFilename));
-    if( !EQUAL(osExt, "xml") )
-        return FALSE;
+    if( STARTS_WITH(poOpenInfo->pszFilename, "/vsizip/") ||
```

Comment:
can't you just remove that check ? Normally if you open a file through /vsizip/ and the like, you should get in poOpenInfo->pabyHeader the header bytes.
And you could then just remove the zip test file and associated test, since there's normally no need to do any particular processing for the /vsizip/ case compared to a regular file

Reply:
Sure, forgot to remove